### PR TITLE
Feat: Add tokens to `SuggestionProductCard`

### PR DESCRIPTION
**Pull request by ArthurTriis1**
Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>

## What's the purpose of this pull request?
This PR applies the new [Theming structure](https://github.com/vtex-sites/base.store/pull/407) to `SuggestionProductCard`: new scoped tokens linked to global ones.

## How to test it?
You can check the story on Storybook.

## Checklist

**Changelog**
- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.
- [ ] Identified the function or parameter in the PR *- If applicable*. E.g., *`useWindowDimensions` hook*.

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*.
- [x] For documentation changes, ping @ carolinamenezes, @ PedroAntunesCosta or @ Mariana-Caetano to review and update.

### CODE REVIEW

**Review by renatamottam — APPROVED** [review]
LGTM!

File: .storybook/mocks/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export { product } from './product'
```
Comment (eduardoformiga) [inline .storybook/mocks/index.ts:1]: Nice! Could you please update it on Nextjs starter as well if it's not too much work? 
I noticed there we have mocks and mock folders.

**Review by eduardoformiga — APPROVED** [review]
LGTM! 👏

File: .storybook/mocks/index.ts (hunk 1 above)
Comment (ArthurTriis1) [reply .storybook/mocks/index.ts:1]: Yeah! I'm opening the PR!

**Pull request merged**